openapi/components/schemas/RiskScore/RiskScoreBlocklist.yaml

```
@@ -0,0 +1,23 @@
+type: object
+description: Risk score blocklist configuration.
+properties:
+  address:
+    $ref: "./RiskScoreBlocklistType.yaml"
```

```suggestion
    $ref: ./RiskScoreBlocklistType.yaml
```

I think double quotes are excessive here